services/contract_create.proto

```diff
@@ -166,15 +166,38 @@ message ContractCreateTransactionBody {
     string memo = 13;
 
     /**
-     * The maximum number of tokens that this contract can be automatically associated 
+     * The maximum number of tokens that this contract can be automatically associated
      * with (i.e., receive air-drops from).
      */
     int32 max_automatic_token_associations = 14;
 
     /**
-     * An account to charge for auto-renewal of this contract. If not set, or set to an 
-     * account with zero hbar balance, the contract's own hbar balance will be used to 
+     * An account to charge for auto-renewal of this contract. If not set, or set to an
+     * account with zero hbar balance, the contract's own hbar balance will be used to
      * cover auto-renewal fees.
      */
     AccountID auto_renew_account_id = 15;
+
+    /**
+     * ID of the new account or node to which this contract is staking. It can never happen that staked_account_id is
+     * non-null and staked_node_id is non-negative at the same time.
+     */
+    oneof staked_id {
+
+        /**
+         * ID of the account to which this contract is staking. If not staked to an account, the value is null.
+         * The default value is null.
+         */
+        AccountID staked_account_id = 17;
+
+        /**
+         * ID of the node this contract is staked to. If not staked to a node, the value is -1. The default value is -1.
```

Comment:
Since it's in wrapped in a `oneof`, no need to have a sentinel value to indicate it's not staked to a node.

```suggestion
         * ID of the node this contract is staked to.
```

Reply:
Left the default value in place and fixed

Reply:
Again, the default is not -1 it's unset. The comment should remove the default. Same thing for staked_account_id.